tools/cdb2_sqlreplay/CMakeLists.txt

```diff
@@ -18,4 +18,7 @@ target_link_libraries(cdb2_sqlreplay ${libs})
 if(COMDB2_BUILD_STATIC)
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -static-libgcc -static-libstdc++")
 endif()
+
+set_property(TARGET cdb2_sqlreplay PROPERTY CXX_STANDARD 11)
```

Comment:
Wasn't this an issue on IBM and Sun?

Reply:
Yes, it was.  For the time being we're only going to make this available on Linux.